examples/03_connectivity/plot_adhd_spheres.py

```diff
@@ -60,23 +60,23 @@
 
 
 ##########################################################################
-# Compute precision matrices
-from sklearn.covariance import LedoitWolf
-cve = LedoitWolf()
-cve.fit(time_series)
-
+# Compute partial correlation matrix
+from nilearn.connectome import ConnectivityMeasure
+connectivity_measure = ConnectivityMeasure(kind='partial correlation')
+partial_correlation_matrix = connectivity_measure.fit_transform(
+    [time_series])[0]
```

Comment:
Yes, `ConnectivityMeasure` default covariance estimator is LW.
May be I can add a comment on that or explicit `cov_estimator` ?